tests/stress/scopez_server/scopez_stress_test.py

```diff
@@ -0,0 +1,279 @@
+#!/usr/bin/env python3
+# ------------------------------------------------------------------------------
+# stress test scopez_server
+# ------------------------------------------------------------------------------
+# ------------------------------------------------------------------------------
+# Imports
+# ------------------------------------------------------------------------------
+import sys
+import argparse
+import time
+import signal
+import json
+import random
+import base64
+import datetime
+import requests
+from urllib.request import urlopen
+from urllib.request import Request
+# ------------------------------------------------------------------------------
+# Globals
+# ------------------------------------------------------------------------------
+g_run = True
+# ------------------------------------------------------------------------------
+#
+# ------------------------------------------------------------------------------
+def signal_handler(signal, frame):
+    global g_run
+    g_run = False
+# ------------------------------------------------------------------------------
+#
+# ------------------------------------------------------------------------------
+def print_banner():
+    print('+-----------------------------------------------------------------------------+')
+    print('|            SCOPEZ   S E R V E R   S T R E S S   T E S T E R              |')
+    print('+------------+------------+------------+------------+------------+------------+')
+    print('| Req/s      | 200s       | 300s       | 400s       | 500s       | Confs/s    |')
+    print('+------------+------------+------------+------------+------------+------------+')
+# ------------------------------------------------------------------------------
+#
+# ------------------------------------------------------------------------------
+def print_stats_line(a_time_delta_ms, a_num_reqs, a_num_configs, a_results):
+    if '200' not in a_results:
+        a_results['200'] = 0
+    if '300' not in a_results:
+        a_results['300'] = 0
+    if '400' not in a_results:
+        a_results['400'] = 0
+    if '500' not in a_results:
+        a_results['500'] = 0
+    print('| %10.2f | %10d | %10d | %10d | %10d | %10d |' % (
+        (a_num_reqs*1000/a_time_delta_ms),
+        a_results['200'],
+        a_results['300'],
+        a_results['400'],
+        a_results['500'],
+        (a_num_configs*1000/a_time_delta_ms)))
+# ------------------------------------------------------------------------------
+#
+# ------------------------------------------------------------------------------
+def get_rqst(a_host, a_id, a_vectors, a_idx, a_results):
+    l_url = a_host
+    l_v = a_vectors[a_idx]
+    l_headers = None;
```

Comment:
We're not using a_id in this function.  Would it help to set this in the headers for tracking requests, similar to waflz stress?